warehouse/migrations/env.py

```diff
@@ -31,6 +31,7 @@ def run_migrations_offline():
     script output.
     """
     options = context.config.get_section(context.config.config_ini_section)
+    assert options is not None  # type narrowing
```

Comment:
We should probably better handle the case where `options` could be `None` here (raise an exception?) instead of assuming it will always be non-`None`.